src/brutus-module-math/brutus_module_math/tests/simpleExpression_test.py

```diff
@@ -40,6 +40,7 @@ def test_addition(self):
                         a = n1Digit + n2Digit
                         text = self.create_question(prefix, n1Text, n2Text, op)
                         answer = self.create_answer(n1Digit, n2Digit, sym, a)
+                        print(text)
```

Comment:
Don't use print in a unit test. You can add comments to assert statements though:

```
assert x % 2 == 0, "value should be even"
```

Check out http://doc.pytest.org/en/latest/assert.html for more examples.